apps/andi/lib/andi_web/live/edit_live_view.ex

```diff
@@ -2,89 +2,184 @@ defmodule AndiWeb.EditLiveView do
+              <%= error_tag(fp, :contactName) %>
+            </div>
+            <div class="metadata-form__maintainer-email">
+              <%= Form.label(fp, :contactEmail, "Maintainer Email", class: "label label--required") %>
+              <%= Form.text_input(fp, :contactEmail, class: "input") %>
+              <%= error_tag(fp, :contactEmail) %>
+            </div>
+            <div class="metadata-form__release-date">
+              <%= Form.label(fp, :issuedDate, "Release Date", class: "label label--required") %>
+              <%= Form.text_input(fp, :issuedDate, class: "input") %>
+              <%= error_tag(fp, :issuedDate) %>
+            </div>
+            <div class="metadata-form__license">
+              <%= Form.label(fp, :license, "License", class: "label label--required") %>
+              <%= Form.text_input(fp, :license, class: "input") %>
+              <%= error_tag(fp, :license) %>
+            </div>
+            <div class="metadata-form__update-frequency">
+              <%= Form.label(fp, :publishFrequency, "Update Frequency", class: "label label--required") %>
+              <%= Form.text_input(fp, :publishFrequency, class: "input") %>
+              <%= error_tag(fp, :publishFrequency) %>
+            </div>
+            <div class="metadata-form__keywords">
+              <%= Form.label(fp, :keywords, "Keywords", class: "label") %>
+              <%= Form.text_input(fp, :keywords, value: get_keywords(Form.input_value(fp, :keywords)), class: "input") %>
+              <div class="label label--inline">Separated by comma</div>
+            </div>
+            <div class="metadata-form__last-updated">
+              <%= Form.label(fp, :modifiedDate, "Last Updated", class: "label") %>
+              <%= Form.text_input(fp, :modifiedDate, class: "input") %>
+            </div>
+            <div class="metadata-form__spatial">
+              <%= Form.label(fp, :spatial, "Spatial Boundaries", class: "label") %>
+              <%= Form.text_input(fp, :spatial, class: "input") %>
+            </div>
+            <div class="metadata-form__temporal">
+              <%= Form.label(fp, :temporal, "Temporal Boundaries", class: "label") %>
+              <%= Form.text_input(fp, :temporal, class: "input") %>
+              <%= error_tag(fp, :temporal) %>
+            </div>
+            <div class="metadata-form__organization">
+              <%= Form.label(fp, :orgTitle, "Organization", class: "label label--required") %>
+              <%= Form.text_input(fp, :orgTitle, [class: "input input--text", disabled: true]) %>
+              <%= error_tag(fp, :orgTitle) %>
+            </div>
+            <div class="metadata-form__language">
+              <%= Form.label(fp, :language, "Language", class: "label") %>
+              <%= Form.select(fp, :language, [[key: "English", value: "english"], [key: "Spanish", value: "spanish"]], value: get_language(Form.input_value(fp, :language)), class: "select") %>
```

Comment:
I would pull the select options out into a module attribute

Reply:
JMS